management/Makefile

```diff
@@ -1,33 +1,27 @@
+# Edit this section to set the values specific to your deployment
+NAME=<NAME>
+PROJECT=<PROJECT>
+ZONE_OR_REGION=<ZONE_OR_REGION>
 
-# The kname of the context for the management cluster
-# The name of the context for your Kubeflow cluster
-NAME=$(shell yq r ./instance/settings.yaml name)
 MGMTCTXT=$(NAME)
 
 # The URL you want to fetch manifests from
 MANIFESTS_URL=https://github.com/kubeflow/manifests.git/gcp/v2/management@master
 
-PROJECT=$(shell yq r ./instance/settings.yaml project)
-
 # Directory where manifests should be fetched to
 MANIFESTS_DIR=./upstream/management
 
 INSTANCE_DIR=./instance
 
-
-#***********************************************************************************************************************
-# Edit this section to set the values specific to your deployment
-
 .PHONY: set-values
 set-values:
-	set-values:	
-	kpt cfg set ./instance name <NAME>
-	kpt cfg set ./instance location <ZONE_OR_REGION>
-	kpt cfg set ./instance gcloud.core.project <PROJECT_TO_DEPLOY_IN>
+	kpt cfg set ./instance name $(NAME)
```

Comment:
Not using make variables was intentional.  The intent was to make the KptFile the source of truth. Hence the set-values stanza (i.e. calling kpt cfg) is the preferred way of setting the values.

I think if we wanted to dedupe the stanzas we could refactor it so we only set the values in one kpt package and then read it from that kpt package.

Reply:
Even if we refactor the kpt packages into one kpt package there needs to be a way to edit these values. I think it won't be changed.

The main concern at the moment is while deploying management cluster I was thinking like maybe `instance pkg` is for a kubeflow cluster and `upstream/management pkg` is for mgmt cluster. And with this minimal change, at least this won’t happen again with the next person trying to deploy the mgmt cluster.